### PR TITLE
Adds functionality to customize the form tag in FormHelper::postButton()

### PR DESCRIPTION
**Pull request by TheFRedFox**
I really like the postButton method. It doesn't need any javascript to work. (slight hint to the CSP topic) :laughing: 
However there was/is no possibility to customize the form tag, like adding css classes or the method type (`put`, `get`, etc.).

With this PR it would be possible to do so. I am working with this code for a while on my own extended FormHelper and it worked well. :wink: Just added two additional tests.

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -1616,7 +1616,16 @@ public function button($title, array $options = [])
      */
     public function postButton($title, $url, array $options = [])
     {
-        $out = $this->create(false, ['url' => $url]);
+        $formOptions = ['url' => $url];
+        if (isset($options['method'])) {
```
Comment (ADmad) [inline src/View/Helper/FormHelper.php]: The new options `method` and `form` should be stated in the method's docblock.

**Comment by markstory** [comment]
Other than the docs omission, this looks good to me.

**Comment by dereuromark** [comment]
:+1:

**Comment by TheFRedFox** [comment]
Added/Changed the docblock a bit. Is it enough or should it be more precise?

File: src/View/Helper/FormHelper.php [hunk 2]
```diff
@@ -1598,14 +1598,17 @@ public function button($title, array $options = [])
     }
 
     /**
-     * Create a `<button>` tag with a surrounding `<form>` that submits via POST.
+     * Create a `<button>` tag with a surrounding `<form>` that submits via POST in default.
```
Comment (jippi) [inline src/View/Helper/FormHelper.php]: as* ?

Comment (TheFRedFox) [inline src/View/Helper/FormHelper.php]: Oh yeah, "in" seems to be completely wrong. :laughing:

**Comment by markstory** [comment]
Changes look good to me. I restarted the builds as travis bailed the first time around.

**Pull request merged**